src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -385,6 +382,12 @@ def update_existing_check_in(
                 "monitors.checkin.result",
                 tags={**metric_kwargs, "status": "complete"},
             )
+    except UnableToAcquireLock:
+        metrics.incr(
+            "monitors.checkin.result",
+            tags={**metric_kwargs, "status": "failed_checkin_creation_lock"},
+        )
+        logger.debug("failed to acquire lock to create check-in: %s", guid)
```

Comment:
Does `.debug` do string formatting like that? Might be safer to use an fstring and format it before passing. Or just include the guid in `extra={...`

Reply:
mmm I think so. the other lines do it. yeah i was just copy/pasting. can cut a separate PR to improve our logging. this stuff doesn't even get logged in prod regardless